build.gradle

```diff
@@ -26,7 +26,9 @@ dependencies {
 	implementation 'org.springframework.boot:spring-boot-starter-jdbc'
 	implementation 'org.postgresql:postgresql'
 	implementation 'org.springframework.boot:spring-boot-starter-web'
-	developmentOnly 'org.springframework.boot:spring-boot-devtools'
+    implementation 'junit:junit:4.12'
+    implementation 'org.jetbrains:annotations:20.1.0'
```

Comment:
these two dependencies are not required, please remove them. Keep only necessary dependencies in the project as it adds to the size of the deployable artifact.